lib/src/components/Sections/index.vue

```diff
@@ -0,0 +1,1922 @@
+              // and name that has only name of the section which is going to be used for importing the section by using only its name on the host project.
+              section.nameID = section.name;
+              section.name = section.name.split(":")[1];
+            } else if (section.settings) {
+              section.settings = JSON.parse(section.settings);
+            }
+            if (section.id) {
+              views[section.id] = section;
+            } else {
+              views["test"] = section;
+            }
+          });
+          this.$set(this.displayVariations, this.activeVariation.pageName, {
+            name: this.activeVariation.pageName,
+            views: { ...views },
+          });
+          this.selectedVariation = this.activeVariation.pageName;
+          this.loading = false;
+          this.$emit("load", true);
+          this.sectionsPageLastUpdated = res.data.last_updated;
+        } catch (error) {
+          if(error.response.data.error) {
+            this.sectionsError = error.response.data.error
+          } else {
+            this.sectionsError = error.response.data.message
+          }
+          this.loading = false;
+          this.pageNotFound = true;
+          this.$emit("load", false);
+        }
+      }
+    }
+  },
+  watch: {
+    isModalOpen(value) {
+      const body = document.querySelector("body");
+      if(value === true) {
+        body.style.overflow = "hidden";
+      } else {
+        body.style.overflow = "auto";
+      }
+    }
+  },
+  methods: {
+    checkToken() {
+      const auth_code = this.$route.query.auth_code;
+      if (auth_code) {
+        const config = {
+          headers: sectionHeader({}),
+        };
+        const URL =
+          `${this.$sections.serverUrl}/project/${this.$sections.projectId}/token/${auth_code}`;
+        this.$axios
+          .get(URL, config)
+          .then((res) => {
+            const token = res.data.token;
+            const date = new Date();
+            date.setDate(date.getDate() + 7);
+            this.$cookies.set("sections-auth-token", token, {expires: date});
+            this.$nuxt.context.redirect(this.$route.path)
+            this.loading = false;
+          })
+          .catch((err) => {
+            this.loading = false;
+          });
+      }
+    },
+    exportSections() {
+      const dataStr = "data:text/json;charset=utf-8," + encodeURIComponent(JSON.stringify(this.allSections));
+      const dlAnchorElem = document.getElementById('downloadAnchorElem');
+      dlAnchorElem.setAttribute("href",     dataStr     );
+      dlAnchorElem.setAttribute("download", `${this.pageName}.json`);
+      dlAnchorElem.click();
+    },
+    initImportSections() {
+      if (Object.keys(this.displayVariations[this.selectedVariation].views).length > 0) {
+        this.showToast(
+          "Warning",
+          "warning",
+          "Import sections only works for empty pages"
```

Comment:
translations ?

Reply:
shouldn't you also translate the title of the toast?

Reply:
The new toast we are using doesn't have a title